assets/js/atomic/blocks/product-elements/shared/config.tsx

```diff
@@ -23,12 +24,10 @@ export default {
 		html: false,
 	},
 	parent: isExperimentalBuild()
-		? null
+		? undefined
 		: [ '@woocommerce/all-products', '@woocommerce/single-product' ],
 	save,
-	deprecated: [
-		{
-			save() {},
```

Comment:
I am not experienced enough with the codebase/APIs to know if this is safe to remove. Can we get another opinion?

Reply:
Deleting this would cause a regression if someone inserted All Products block before version 3.2.0, never resaved the page, and accessed the page.

In normal cases I would say this is safe to delete; however, an invalid block might break the inner block structure of All Products. Keeping this here doesn't seem awfully annoying to us?